js/busfinder.js

```diff
@@ -1,26 +1,15 @@
+//added require lines for views used
+var ContentView = require('./views/ContentView'),
+    HomeView = require('./views/HomeView'),
+    MapView = require('./views/MapView');
+
```

Comment:
This is the right idea, but the end goal is that everything in busfinder.js will be removed and the file deleted, so this isn't necessary.